tests/e2e/page.html

```diff
@@ -20,5 +20,16 @@ <h1>Test Page</h1>
       id="custom-filter"
       style="display: block; height: 100px; background: blue"
     ></div>
+
+    <!-- Scriptlets with web accessible resources -->
+    <script defer>
+      void (async function () {
+        const res = await fetch('ads.js');
+        const div = document.createElement('div');
```

Comment:
The div can be static and put in HTML code, or is it important to be added dynamically?

Reply:
I believe it's a reliable and clean way to notify that the request is finished and to deliver the response body. This way, we can just use `waitFor` instead of relying on complex function calls.